config/boards/shields/the_paintbrush/the_paintbrush_left.conf

```diff
@@ -1,20 +1,20 @@
-# Copyright (c) 2021 Mike "KemoNine" Crosson
-# SPDX-License-Identifier: MIT
-
-# Tune combos to allow proper ARTSEY support
-CONFIG_ZMK_COMBO_MAX_COMBOS_PER_KEY=16
-CONFIG_ZMK_COMBO_MAX_KEYS_PER_COMBO=8
-CONFIG_ZMK_COMBO_MAX_PRESSED_COMBOS=8
-
-# Tune bluetooth profiles for quick select
-CONFIG_BT_MAX_CONN=6
-
-# Enable display (layer in use is helpful)
-CONFIG_ZMK_DISPLAY=y
-
-# Turn on deep sleep
-CONFIG_ZMK_SLEEP=y
-CONFIG_ZMK_IDLE_SLEEP_TIMEOUT=900000
-
-# Set screen blanking to 15s instead of 30s
-#CONFIG_ZMK_IDLE_TIMEOUT=15000
+# Copyright (c) 2021 Mike "KemoNine" Crosson
+# SPDX-License-Identifier: MIT
+
+# Tune combos to allow proper ARTSEY support
+CONFIG_ZMK_COMBO_MAX_COMBOS_PER_KEY=32
```

Comment:
Please update this definition for all of the available boards. The main artsey.dtsi comments and other info needs to be updated to reflect this increase.

It's required so we gotta update all the places.